CreateRestaurantFunction.cs

```diff
@@ -18,28 +18,29 @@ public static class CreateRestaurantFunction
         {
             log.Info("CreateRestaurant function processed a request.");
 
-            string strMongoDBAtlasUri = System.Environment.GetEnvironmentVariable("MongoDBAtlasURI");
-            log.Info($"Atlas connection string is { strMongoDBAtlasUri}");
+            var strMongoDBAtlasUri = System.Environment.GetEnvironmentVariable("MongoDBAtlasURI");
+            log.Info($"Atlas connection string is {strMongoDBAtlasUri}");
 
-            MongoUrl mongoUrl = new MongoUrl(strMongoDBAtlasUri);
+            var mongoUrl = new MongoUrl(strMongoDBAtlasUri);
             var settings = MongoClientSettings.FromUrl(mongoUrl);
             //for more on why we're using ServerSelectionTimeout, read https://scalegrid.io/blog/understanding-mongodb-client-timeout-options/
-            settings.ServerSelectionTimeout = new System.TimeSpan(0, 0, 5);
+            settings.ServerSelectionTimeout = TimeSpan.FromSeconds(5);
 
             var client = new MongoClient(settings);           
```

Comment:
I'm not sure what the run profile is for AzureFunctions, But this is going to spin up a new connection pool everytime.  This will have pretty poor performance characteristics when under heavy load. 

It might be that the instance is left running and, due to how the driver handles using the same underlying cluster for the same settings, might not be a big deal. However, you should probably confirm this and, it might even make sense to put this in a singleton somewhere to hammer home that we don't want to be recreating this everytime.